modules/concept-docs/pages/collections.adoc

```diff
@@ -1,11 +1,31 @@
-= Collections & Scope
-:description: pass:q[Available as a _Developer Preview_ in Couchbase Server 6.6.]
+= Collections & Scopes
+:description: pass:q[Fully supported in Couchbase Server 7.0.]
 :page-topic-type: concept
-:page-aliases:
+:nav-title: Collections
 
 [abstract]
 {description}
 
- 
+The Collections feature in Couchbase Server is fully implemented in the 3.2 API version of the Couchbase SDK.
 
-Please see the xref:6.6@server:developer-preview:collections/collections-overview.adoc[Collections Overview documents] in the Server docs.
+Information on _Collections_ can be found in the xref:7.0@server:learn:data:scopes-and-collections.adoc[server docs].
+
+== Using Collections & Scopes
+
+Access a non-default collection, in the default scope, with:
+
+[source,ruby]
+----
+include::example$collections.rb[tag=collections_1,indent=0]
+----
+
+And for a non-default scope:
+
+[source,ruby]
+----
+include::example$collections.rb[tag=collections_2,indent=0]
+----
+
+== Further Reading
+
+To see Collections in action, take a look at our xref:howtos:working-with-collections.adoc[Collections-enabled Travel Sample page].
```

Comment:
Hmm, this isn't what we're doing anymore is it? We should point this to sample-application.adoc instead.

(Eeek, I've written a similar line before... it's maybe ok-for-now, as there'll be just one layer of indirection as the working-with-collections page will then point to travel-sample, but we should fix, today if poss, or this week)

Reply:
Think I will comment this out, we don't even have a travel-sample Ruby app yet >_<